plugins/pipelines/app/decorators/stage_decorator.rb

```diff
@@ -0,0 +1,9 @@
+Stage.class_eval do
+  serialize :next_stage_ids, Array
+
+  def next_stage
```

Comment:
this function is used for that button on deploy page, and only expects one stage. So didn't want to change it for this PR.